kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -28,9 +28,11 @@ def subsample_fourier(x, k):
         The input tensor periodized along the next to last axis to yield a
         tensor of size x.shape[-2] // k along that dimension.
     """
+    complex_check(x)
```

Comment:
Are these changes intended for this PR or is this a stray commit from the R2C PR?

Reply:
Intended for this PR.